terracumber/terraformer.py

```diff
@@ -66,8 +66,8 @@ def inject_repos(self, custom_repositories_json):
                     node_mu_repos = repos.get(node, None)
                     replacement_list = ["additional_repos = {"]
                     for name, url in node_mu_repos.items():
-                        replacement_list.append('\n\t"{}"="{}",'.format(name, url))
-                    replacement_list.append("\n}")
+                        replacement_list.append(f'\n{" " * 4}{name} = "{url}"')
```

Comment:
This just seems a format refactor, so it doesn't really match the title of the PR right? If so, I would re-title this PR. Aside that, LGTM.

Reply:
Ok, done.